wlog/logentry.go

```diff
@@ -79,9 +79,6 @@ func (m *MapValueEntries) StringMapValue(key string, values map[string]string) {
 }
 
 func (m *MapValueEntries) AnyMapValue(key string, values map[string]interface{}) {
-	if len(values) == 0 {
```

Comment:
This is change is unrelated but this statement is identical to the one below it so I deleted it.